content/en/docs/tasks/debug-application-cluster/troubleshooting.md

```diff
@@ -55,41 +51,41 @@ If there aren't any existing questions that help, please [ask a new one](http://
 
 ### Slack
 
-The Kubernetes team hangs out on Slack in the `#kubernetes-users` channel. You
-can participate in discussion with the Kubernetes team [here](https://kubernetes.slack.com).
-Slack requires registration, but the Kubernetes team is open invitation to
-anyone to register [here](http://slack.kubernetes.io). Feel free to come and ask
-any and all questions.
+Many people from the Kubernetes community hang out on Kubernetes Slack in the `#kubernetes-users` channel.
+Slack requires registration; you can [request an invitation](https://slack.kubernetes.io),
+and registration is open to everyone). Feel free to come and ask any and all questions.
+Once registered, access the [Kubernetes organisation in Slack](https://kubernetes.slack.com)
+via your web browser or via Slack's own dedicated app.
 
-Once registered, browse the growing list of channels for various subjects of
+Once you are registered, browse the growing list of channels for various subjects of
 interest. For example, people new to Kubernetes may also want to join the
-`#kubernetes-novice` channel. As another example, developers should join the
-`#kubernetes-dev` channel.
+[`#kubernetes-novice`](https://kubernetes.slack.com/messages/kubernetes-novice) channel. As another example, developers should join the
+[`#kubernetes-dev`](https://kubernetes.slack.com/messages/kubernetes-dev) channel.
 
-There are also many country specific/local language channels. Feel free to join
+There are also many country specific / local language channels. Feel free to join
 these channels for localized support and info:
 
-- China: `#cn-users`, `#cn-events`
-- Finland: `#fi-users`
-- France: `#fr-users`, `#fr-events`
-- Germany: `#de-users`, `#de-events`
-- India: `#in-users`, `#in-events`
-- Italy: `#it-users`, `#it-events`
-- Japan: `#jp-users`, `#jp-events`
-- Korea: `#kr-users`
-- Netherlands: `#nl-users`
-- Norway: `#norw-users`
-- Poland: `#pl-users`
-- Russia: `#ru-users`
-- Spain: `#es-users`
-- Sweden: `#se-users`
-- Turkey: `#tr-users`, `#tr-events`
+- China: [`#cn-users`](https://kubernetes.slack.com/messages/cn-users`), [`#cn-users`](https://kubernetes.slack.com/messages/cn-events`)
+- Finland: [`#fi-users`](https://kubernetes.slack.com/messages/fi-users)
+- France: [`#fr-users`](https://kubernetes.slack.com/messages/fr-users`), [`#fr-users`](https://kubernetes.slack.com/messages/fr-events`)
+- Germany: [`#de-users`](https://kubernetes.slack.com/messages/de-users`), [`#de-users`](https://kubernetes.slack.com/messages/de-events`)
+- India: [`#in-users`](https://kubernetes.slack.com/messages/in-users`), [`#in-users`](https://kubernetes.slack.com/messages/in-events`)
```

Comment:
The second channel string should be something like: `[#fr-events]`?

Reply:
Ah yep